lang/python/python-pyotp/Makefile

```diff
@@ -0,0 +1,40 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-pyotp
+PKG_VERSION:=2.2.7
+PKG_RELEASE:=1
+
+PKG_SOURCE:=pyotp-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://files.pythonhosted.org/packages/source/p/pyotp
+PKG_HASH:=be0ffeabddaa5ee53e7204e7740da842d070cf69168247a3d0c08541b84de602
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(BUILD_VARIANT)-pyotp-$(PKG_VERSION)
+
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+PKG_MAINTAINER:=Josef Schlehofer <josef.schlehofer@nic.cz>
+
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+PKG_UNPACK:=$(HOST_TAR) -C $(PKG_BUILD_DIR) --strip-components=1 -xzf $(DL_DIR)/$(PKG_SOURCE)
```

Comment:
Minor comment: Since there is only a Python 3 package, there is no need to redefine `PKG_BUILD_DIR` and `PKG_UNPACK`. Leaving it like this doesn't do any harm either.

Reply:
In this case PKG_BUILD_DIR needs to be there.

In build_dir folder, there are two folders
pyotp-2.2.7 which has necessary files, but it expects files in python-pyotp-2.2.7.

```
./setup.py  install --prefix="/usr" --root="/home/pepe/work/turrisos40/mox-hbd/build/build_dir/target-aarch64_cortex-a53_musl/python-pyotp-2.2.7/ipkg-install" --single-version-externally-managed ;
/home/pepe/work/turrisos40/mox-hbd/build/staging_dir/hostpkg/bin/python3.7: can't open file './setup.py': [Errno 2] No such file or directory
make[2]: *** [Makefile:47: /home/pepe/work/turrisos40/mox-hbd/build/build_dir/target-aarch64_cortex-a53_musl/python-pyotp-2.2.7/.built] Error 2
```

Reply:
Also it seems there must be PKG_UNPACK as well.

Reply:
Thank you for your suggestion. Removed, but I need to change PKG_NAME otherwise it complained about a few errors. :-)